test/Decompressor.js

```diff
@@ -105,32 +105,32 @@ describe('Decompressor', function () {
         it('should decompress without calldata', async function () {
             const { decompressorExt } = await loadFixture(initContractsWithDict);
             const decompressedCalldata = await ethers.provider.call({
-                to: decompressorExt.address,
+                to: decompressorExt,
```

Comment:
Why no `await x.getAddress()` here?

Reply:
From what I've seen in `hardhat` release notes, it's possible to pass contracts into transactions without first getting their address
<img width="461" alt="image" src="https://github.com/1inch/calldata-compressor/assets/94478708/4b496eb5-4d5c-47bd-8f53-886fea7a90ce">

Reply:
Cool, I suspected this :)